src/peers/peers.tsx

```diff
@@ -5,9 +5,12 @@ import { peersContainerState } from './peersContainer'
 type PeersProps = peersContainerState
 
 const Peers: React.SFC<PeersProps> = ({ peers }) => {
-  const peersData = peers.join(', ')
+  if(peers.length !== 0) {
+    const peersData = peers.join(', ')
+    return <span className="label label--small">{peersData}</span>
+  }
```

Comment:
If you can have an `else` clause it is better to use it rather than fall-though like this. I then don't need to check whether the code _inside_ the if statement is returning to know what the execution flow is. A minor point.